bin/prokka

```diff
@@ -621,7 +621,8 @@ if ($rfam) {
     my $num_ncrna = 0;
     my $tool = "Infernal:".$tools{'cmscan'}->{VERSION};
     my $icpu = $cpus || 1;
-    my $cmd = "cmscan --rfam --cpu $icpu -E $evalue --tblout /dev/stdout -o /dev/null --noali $cmdb \Q$outdir/$prefix.fna\E";
+    my $dbsize = $total_bp * 2 / 1000000;
+    my $cmd = "cmscan -Z $dbsize --cut_ga --rfam --nohmmonly --cpu $icpu --tblout /dev/stdout -o /dev/null --noali $cmdb \Q$outdir/$prefix.fna\E";
```

Comment:
```suggestion
    my $cmd = "cmscan -Z $dbsize --cut_ga --rfam --nohmmonly --cpu $icpu --fmt 2 --tblout /dev/stdout -o /dev/null --noali $cmdb \Q$outdir/$prefix.fna\E";
```

Reply:
Addressed in 717be15. Relevant output now looks like this.

```
chrom	Infernal:1.1	misc_RNA    19391	19533	60.9	+	.	ID=GGDGEHMO_00018;Note="Salmonella enterica Typhi npcRNA 44";accession=RF01830;inference=COORDINATES:profile:Infernal:1.1;locus_tag=GGDGEHMO_00018;product=StyR-44
chrom	Infernal:1.1	misc_RNA    22999	23105	101.6	+	.	ID=GGDGEHMO_00021;Note="Yersinia sRNA 224";accession=RF02770;inference=COORDINATES:profile:Infernal:1.1;locus_tag=GGDGEHMO_00021;product=Ysr224
chrom	Infernal:1.1	misc_RNA    30017	30134	121.1	+	.	ID=GGDGEHMO_00028;Note="Spot 42 RNA";accession=RF00021;inference=COORDINATES:profile:Infernal:1.1;locus_tag=GGDGEHMO_00028;product=Spot_42
chrom	Infernal:1.1	misc_RNA    31270	31574	322.2	+	.	ID=GGDGEHMO_00030;Note="Yersinia sRNA 186/sR026/CsrC";accession=RF02767;inference=COORDINATES:profile:Infernal:1.1;locus_tag=GGDGEHMO_00030;product=Ysr186_sR026_CsrC
chrom	Infernal:1.1	misc_RNA    53847	54101	202.5	+	.	ID=GGDGEHMO_00051;Note="Yersinia sRNA 141";accession=RF02675;inference=COORDINATES:profile:Infernal:1.1;locus_tag=GGDGEHMO_00051;product=Ysr141
chrom	Infernal:1.1	misc_RNA    104578	104684	93.1	+	.	ID=GGDGEHMO_00102;Note="Yersinia sRNA 224";accession=RF02770;inference=COORDINATES:profile:Infernal:1.1;locus_tag=GGDGEHMO_00102;product=Ysr224
chrom	Infernal:1.1	misc_RNA    115332	115396	57.2	-	.	ID=GGDGEHMO_00113;Note="ClpQY promoter";accession=RF02401;inference=COORDINATES:profile:Infernal:1.1;locus_tag=GGDGEHMO_00113;product=ClpQY_promoter
chrom	Infernal:1.1	misc_RNA    157074	157301	103.5	+	.	ID=GGDGEHMO_00146;Note="Cobalamin riboswitch";accession=RF00174;inference=COORDINATES:profile:Infernal:1.1;locus_tag=GGDGEHMO_00146;product=Cobalamin
chrom	Infernal:1.1	misc_RNA    162458	162600	60.9	+	.	ID=GGDGEHMO_00152;Note="Salmonella enterica Typhi npcRNA 44";accession=RF01830;inference=COORDINATES:profile:Infernal:1.1;locus_tag=GGDGEHMO_00152;product=StyR-44
chrom	Infernal:1.1	misc_RNA    174576	174876	180.5	+	.	ID=GGDGEHMO_00164;Note="Soft rot Enterobacteriaceae Rev 39 5'UTR";accession=RF02744;inference=COORDINATES:profile:Infernal:1.1;locus_tag=GGDGEHMO_00164;product=Rev39_5p_UTR
```